kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/ResourceSelection.vue

```diff
@@ -491,6 +481,17 @@
         loadingMore,
       } = useQuizResources({ topicId, practiceQuiz: selectPracticeQuiz.value });
 
+      // TODO let's not use text for this
+      const viewMoreButtonState = computed(() => {
+        if (loadingMore.value) {
+          return ViewMoreButtonStates.LOADING;
```

Comment:
Oh I did this to remove the `loadingMoreState` prop from the `ContentCardList`, because this `loadingMoreState` can be reflected through this viewMoreButtonState, and we dont need another variable for this. So its not adding a new loading state per se, just relocating the `loadingMoreState` prop that was removed from the `ContentCardList`